capstone-frontend/src/components/BookDescriptionOverlay.jsx

```diff
@@ -6,7 +6,7 @@ import { Popover, OverlayTrigger } from 'react-bootstrap';
 // When referenced as a component in the overlay prop, the Popover always appears at the
 // top left of the screen even when there is a predefined placement.
 const popoutDescription = (book) => {
-  let description = book.description || "";
+  let description = (typeof book !== 'undefined') ? book.description : "";
```

Comment:
In what cases is the book type `undefined`?

Reply:
This check resolves issues @antoniolinhart and I were having with the Popover Overlay inside the SearchBookModal.

Reply:
The search modal was send an undefined prop causing an error.